test/test_evaluation/test_train_evaluator.py

```diff
@@ -545,14 +545,16 @@ def side_effect(self, *args, **kwargs):
         self.assertEqual(pipeline_mock.predict_proba.call_count, 36)
 
     @unittest.mock.patch.object(TrainEvaluator, '_loss')
-    def test_file_output(self, loss_mock):
+    @unittest.mock.patch.object(TrainEvaluator, '_get_model')
```

Comment:
This is because I added this [line](https://github.com/automl/auto-sklearn/blob/5876fb61a905cb6ceff0000a758985dd9b29b333/autosklearn/evaluation/abstract_evaluator.py#L270) as it was a mypy requirement.